PaulasWeatherApp/PaulasWeatherApp/View/HourlyWeatherDetailsCell.swift

```diff
@@ -0,0 +1,50 @@
+//
+//  HourlyWeatherDetailsCell.swift
+//  PaulasWeatherApp
+//
+//  Created by Stef, Paula on 8/10/22.
+//
+
+import UIKit
+
+class HourlyWeatherDetailsCell: UICollectionViewCell {
+    
+    let titleLabel = UILabel()
+    let descriptionLabel = UILabel()
```

Comment:
Do you need these two public?

Reply:
Yes, I use them in HourlyWeatherDetailsViewController to set different values for details. 
<img width="854" alt="Screen Shot 2022-08-11 at 4 20 48 PM" src="https://user-images.githubusercontent.com/97982936/184143438-42720324-a913-463a-80ff-716ddeccac64.png">

Reply:
Should I send the detail to HourlyWeatherDetailsCell instead?